<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerMainMenu</class>
 <widget class="QWidget" name="CustomerMainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#CustomerMainMenu{
	background-color: rgb(39, 44, 54);
}

QWidget#widget{
	border-radius: 10px;
	background-color: rgba(37, 55, 67, 90);
}

QLineEdit{
	min-height: 45px;
	border-radius: 20px;
	background-color: #FFFFFF;
	padding-left: 20px;
	color: rgb(140, 140, 140);
}

QLineEdit:hover{
	border: 2px solid rgb(139, 142, 139);
}

QPushButton{
	min-height: 45px;
	border-radius: 20px;
	background-color: rgb(44, 66, 81);
	color: #FFFFFF;
}

QPushButton:hover{
	border: 2px solid rgb(255, 255, 255);
}

QCheckBox{
	font-size: 10px;
	color: #FFFFFF;
}

QLabel{
	color: rgb(95, 94, 108);
}


</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QScrollArea" name="invitation_sa">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>500</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
QScrollArea{
	margin-left: 20px;
	margin-top: 25px;
	background-color: rgb(39, 44, 54);
	border-radius: 20px;
}



QLineEdit{
	margin-left: 50px;
	min-height: 45px;
	border-radius: 20px;
	background-color: #FFFFFF;
	padding-left: 20px;
	color: rgb(31, 53, 94);
}

QLineEdit:hover{
	border: 2px solid rgb(139, 142, 139);
}

QPushButton{
	margin-left: 60px;
	margin-right: 60px;
	min-height: 45px;
	border-radius: 20px;
	background-color: rgb(44, 66, 81);
	color: #FFFFFF;
}

QPushButton:hover{
	border: 2px solid rgb(255, 255, 255);
}

QCheckBox{
	font-size: 10px;
	color: #FFFFFF;
}

QLabel{
	color: rgb(95, 94, 108);
}


</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="widget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>150</width>
        <height>486</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
	margin-up: 100px;
	margin-down: 200px;
}

QLineEdit{
	margin-left: 50px;
	min-height: 45px;
	border-radius: 20px;
	background-color: #FFFFFF;
	padding-left: 20px;
	color: rgb(31, 53, 94);
}

QLineEdit:hover{
	border: 2px solid rgb(139, 142, 139);
}

QPushButton{
	margin-left: 60px;
	margin-right: 60px;
	min-height: 45px;
	border-radius: 20px;
	background-color: rgb(44, 66, 81);
	color: #FFFFFF;
}

QPushButton:hover{
	border: 2px solid rgb(255, 255, 255);
}

QCheckBox{
	font-size: 10px;
	color: #FFFFFF;
}

QLabel{
	color:rgb(255, 255, 255);
}


</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="menu_frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>450</width>
       <height>0</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame#menu_frame{
	margin: 30px;
	border-radius: 20px;
	background-color: rgba(45, 65, 72, 120);
}

QLineEdit{
	margin-left: 10px;
	min-height: 45px;
	border-radius: 20px;
	background-color: #FFFFFF;
	padding-left: 20px;
	color: rgb(31, 53, 94);
}

QLineEdit:hover{
	border: 2px solid rgb(139, 142, 139);
}

QPushButton{
	margin-left: 20px;
	margin-right: 20px;
	min-height: 45px;
	border-radius: 20px;
	background-color: rgb(44, 66, 81);
	color: #FFFFFF;
}

QPushButton:hover{
	border: 2px solid rgb(255, 255, 255);
}

QCheckBox{
	font-size: 10px;
	color: #FFFFFF;
}

QLabel{
	color: rgb(95, 94, 108);
}


</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255)</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Ana Menü&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="new_room_btn">
        <property name="text">
         <string>Yeni Oda Oluştur</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame{
	border-radius: 20px;
}
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="room_id_le">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>45</height>
            </size>
           </property>
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Oda ID</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="join_btn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>45</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	margin-left: 0px;
	margin-right: 10px;
	min-weight: 100px;
	min-height: 45px;
	border-radius: 20px;
	background-color: rgb(44, 66, 81);
	color: #FFFFFF;
}</string>
           </property>
           <property name="text">
            <string>Bir Odaya Gir</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exit_btn">
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Çıkış</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
